Automattic/jetpack
# Remove getIconColor from Paywall block icon paths

Resolves an issue where the Paywall block appears as Jetpack green in the mobile editor. Removes color properties from the Paywall block SVG itself in favor of setting this color for the web editor in the [block's settings](
https://github.com/Automattic/jetpack/blob/ab0eb7095fd08862c9cdd8db6f74a0f739494c50/projects/plugins/jetpack/extensions/blocks/paywall/index.js#L35C1-L35C1). If these code changes have been done correctly, there should be no modification to the Paywall icon appearance in the Jetpack web editor. On mobile, the icon should not appear as Jetpack green within the editor. 

Fixes `gutenberg-mobile#6099`
* https://github.com/wordpress-mobile/gutenberg-mobile/issues/6099

## Proposed changes:

For the Paywall block SVG icon:
* Remove `fill="none"` on the parent SVG element
* Remove the use of `fill={ getIconColor() }` from the child Path elements

### Other information:


## Jetpack product discussion


* https://github.com/Automattic/jetpack/pull/32553

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:






0. Enable Jetpack beta features so that beta blocks are available
1. Add a Paywall block 
2. Verify that the correct Paywall block icon appears as Jetpack green
3. Open the same post on mobile app with these local changes
4. Verify that the Paywall icon is not green
